akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbJournal.scala

```diff
@@ -20,9 +20,15 @@ import com.typesafe.config.Config
  *
  * Journal backed by a local LevelDB store. For production use.
  */
-private[persistence] class LeveldbJournal(val config: Config) extends AsyncWriteJournal with LeveldbStore {
+private[persistence] class LeveldbJournal(cfg: Config) extends AsyncWriteJournal with LeveldbStore {
   import LeveldbJournal._
 
+  def this() = this(null)
```

Comment:
@johanandren I do not understand your point. Could you give me a hint or sample?